pkg/scanners/trivy/trivy.go

```diff
@@ -89,11 +91,26 @@ func main() {
 	flag.Parse()
 	ctx := context.Background()
 
-	if err := logger.Configure(); err != nil {
+	var err error
+
+	if err = logger.Configure(); err != nil {
 		fmt.Fprintln(os.Stderr, "error setting up logger:", err)
 		os.Exit(generalErr)
 	}
 
+	logger, err := zap.NewProduction()
```

Comment:
let's add a comment that we are adding this because trivy logs are not json encoded and we are doing this for json output